### DISCUSSION TITLE
Its been 3 days and my recommendations have not changed!

### DISCUSSION THREAD

**Discussion by Pagi-Maula** [community/community#14532] [Other Feature Feedback, Questions, & Ideas]
I love this new feature of github. However, the featured repositories have not yet replaced by new once!

**Comment by ar1ja** [comment]
- Are people you follow doing anything?
- Do any projects you have starred have any new released?
- Are you starring enough topics and projects?
- Are you being active on GitHub?
- Are you following enough *(active)* people?

**Reply by ar1ja** [reply]
> No one following No I don't No, I am just joined now hopefully active Yeah [https://www.skyoceanlogistics.net/shipping-and-logistics-companies-in-uae.php](url)

exactly, this is why you don't see any updates, once you are able to say "yes" to all or most of those
questions you will see the updates.

**Reply by Pagi-Maula** [reply]
Thanks for the help!
